package.json

```diff
@@ -21,22 +21,21 @@
         "test:start": "jest --watch"
     },
     "devDependencies": {
-        "@types/enzyme": "2.8.1",
+        "@types/enzyme": "~2.5.38",
```

Comment:
Yes -- I'm lining everything up with React 15.4.  The newer enzyme types are incompatible with the older react types.